src/styles/main.scss

```diff
@@ -43,7 +43,6 @@
 @import 'global/forms/labels';
 @import 'global/forms/fields';
 @import 'global/forms/forms';
-@import 'global/forms/formly';
```

Comment:
If you remove the import of `formly.scss` shouldn't the file `formly.scss` be removed as well? Currently it contains unimported styles.